### PR TITLE
Designs of landing rules and auth pages with routing

### PR DESCRIPTION
**Pull request by Developers-codz**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
-        >
-          Learn React
-        </a>
-      </header>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Landing />} />
+        <Route path="/rules" element={<Rules />} />
+        <Route path="/signup" element={<Signup />} />
+        <Route path="/login" element={<Login />} />
```
Comment (shivanigangadharan) [inline src/App.js:14]: You can ommit the usage of `/` as it's not reqd in react router v6

File: src/pages/landing/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { Link } from 'react-router-dom'
+// import HeroImg from "../../assets/svg/heroImg.svg"
+import painting from "../../assets/images/painting.jpg"
+/* eslint-disable */
+export const Landing = () => {
+  return (
+    <main>
+      <div className="hero-wrapper">
+        <div className="left-pane">
+       {/* <HeroImg /> */}
```
Comment (shivanigangadharan) [inline src/pages/landing/Landing.jsx:11]: Can remove comments wherever not reqd

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, can merge this

File: src/components/navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+    <div className="mr-auto page-link-wrapper font3">
+      <ul className="style-list-none">
+        <li className="inline-block">
+          <a href="index.html" className="page-link decor-none active-item"
+            >Home</a
+          >
+        </li>
+        <li className="inline-block">
+          <a href="dashboard.html" className="page-link decor-none">dashboard</a>
```
Comment (vishalg8454) [inline src/components/navbar/Navbar.jsx:18]: You can replace <a> tags with Link from React Router

**Review by vishalg8454 — APPROVED** [review]
Looks fine to merge for now.

File: src/App.js (hunk 1 above)
Comment (Developers-codz) [reply src/App.js:14]: Okay

File: src/pages/landing/Landing.jsx (hunk 2 above)
Comment (Developers-codz) [reply src/pages/landing/Landing.jsx:11]: Yeah sure, will do it in next pr

File: src/components/navbar/Navbar.jsx (hunk 3 above)
Comment (Developers-codz) [reply src/components/navbar/Navbar.jsx:18]: Yep, Will be doing when I work on that page

**Pull request merged**